--- libdca-0.0.5/configure.ac.orig	2007-04-10 14:28:07.000000000 +0200
+++ libdca-0.0.5/configure.ac	2010-11-28 13:12:23.514670272 +0100
@@ -27,13 +27,6 @@
     TRY_CFLAGS="$OPT_CFLAGS -Wall"
     AC_TRY_CFLAGS([$TRY_CFLAGS $CFLAGS],[OPT_CFLAGS=$TRY_CFLAGS])
 
-    dnl -O3
-    changequote(<<,>>)
-    TRY_CFLAGS=`echo "$OPT_CFLAGS $CFLAGS"|sed "s/-O[0-9]*//g"`
-    changequote([,])
-    TRY_CFLAGS="$TRY_CFLAGS -O3"
-    AC_TRY_CFLAGS([$TRY_CFLAGS],[OPT_CFLAGS=$TRY_CFLAGS; CFLAGS=""])
-
     AC_ARG_ENABLE([debug],
 	[  --enable-debug          debug mode configuration])
     if test x"$enable_debug" = x"yes"; then
@@ -44,21 +37,6 @@
 	AC_TRY_CFLAGS([$TRY_CFLAGS $CFLAGS],[OPT_CFLAGS=$TRY_CFLAGS])
     fi
 
-    dnl arch-specific flags
-    case "$host" in
-    i?86-* | k?-*)
-	case "$host" in
-	i386-*) TRY_CFLAGS="$OPT_CFLAGS -mtune=i386";;
-	i486-*) TRY_CFLAGS="$OPT_CFLAGS -mtune=i486";;
-	i586-*) TRY_CFLAGS="$OPT_CFLAGS -mtune=pentium";;
-	i686-*) TRY_CFLAGS="$OPT_CFLAGS -mtune=pentiumpro";;
-	k6-*)   TRY_CFLAGS="$OPT_CFLAGS -mtune=k6";;
-	esac
-	AC_TRY_CFLAGS([$TRY_CFLAGS $CFLAGS],[OPT_CFLAGS=$TRY_CFLAGS]);;
-    sparc-* | sparc64-*)
-	TRY_CFLAGS="$OPT_CFLAGS -mtune=ultrasparc"
-	AC_TRY_CFLAGS([$TRY_CFLAGS $CFLAGS],[OPT_CFLAGS=$TRY_CFLAGS]);;
-    esac
 elif test x"$CC" = x"tendracc"; then
     dnl TenDRA portability checking compiler
     TENDRA=yes
